package.xml

```diff
@@ -6,4 +6,14 @@
   <description>Package containing various utility types and functions for C++</description>
```

Comment:
It would be great to add some more to the description here to start a list of the things contained in this package. And also to have a link to documentation on how to use these macros.

Reply:
Ok, is this the sort of thing we want in the package.xml, or in the README?

Reply:
The README can and should be much more verbose. At the package.xml level it needs at least a high level summary of what's int the package. "various utilitity types and functions" doesn't tell you anything. This high level summary will be the first thing that people read, and will be redistributed into things like debian package descriptions.

Reply:
Personally, I never read this description, but giving some examples of what is in it would be useful.

For an example README, see `osrf_testing_tools_cpp` for what I consider to be a decent example:

https://github.com/osrf/osrf_testing_tools_cpp/blob/master/README.md

Reply:
Have you seen the latest updated version? I added a README with information on usage - do you have specific comments on that?

Reply:
No, I think the current state is ok, just adding to what @tfoote said.

Reply:
Ok, thanks!